test/amount-test.js

```diff
@@ -1182,11 +1187,11 @@ describe('Amount', function() {
 
   describe('amount limits', function() {
     it ('max JSON wire limite', function() {
-      assert.strictEqual(Amount.bi_xns_max.toString(), '9000000000000000000');
+      assert.strictEqual(Amount.bi_xns_max.toString(), '100000000000000000');
```

Comment:
How is that possible? I thought this was the total number of XRP drops created at the beginning (100 billion XRP * 1 million drops/XRP = 1e11 * 1e6 = 1e17).